### DISCUSSION TITLE
ATSAS OLIGOMER output

### DISCUSSION THREAD

**Discussion by ckos23** [biosaxs-com/atsas-community#1965] [General]
Hi,

I am confused about the output I get from running oligomer. 
I ran SAXS data for an enzyme and it has a molar weight of ~30kDa. 
I only gave two states (monomer and dimer). 
But the output shows 5 Mws and Volume fractions. (Real oligomer weights 12780 4534 8663 8825 40415)
I would like to know,
1. How does Oligomer calculate the Mw? 
2. Why do I get 5 volume fractions for two states?
3. Can I change the Option from 0 to 1?

Total output

> 29-Oct-2024 16:36:49
> Option 0: a set of form-factors and several sets of experimental data
> Form-factor file ff.dat
> Real oligomer weights 12780 4534 8663 8825 40415
> Oligomer radii of gyration 35.63 17.69 27.88 24.91 0.00
> Experimental data file Sample1_Wt_post_1mg_A1_subtr.dat Range of Scattering angle: 0.01 0.50
> Using non-negativity condition
> Using constant as additional component
> Output file Sample1_Wt_post_1mg_A1_subtr.fit
> Chi^2 Volume fractions +- errors
> --------------------------------------------------------------------------------
> ================================================================================
> Sample1_Wt_post_1mg_ 5.76 11802. 33.9 0.752+-0.023 0.000+-0.000 0.000+-0.000 0.248+-0.022 0.00000E+00+- 0.00000E+00

Thank you!

**Comment by franke-biosaxs** [comment]
Hi. I am also confused. Could you please start from the beginning and show your steps? How did you assemble the inputs for oligomer, how did you run it etc. Thanks.

**Reply by ckos23** [reply]
![atsas](https://github.com/user-attachments/assets/15c870ed-dd7b-4fc1-9387-31f64c913fa9)
This are the options I chose. 
I added the PDBs of the monomer and the dimer in a zipped file and also gave my data file. I don't understand the output I got though

**Reply by franke-biosaxs** [reply]
Ah. ATSAS Online. Different beast. Then it depends on the contents of the .zip. Going out on a very thin limb here:  are you maybe on a Mac?

**Reply by ckos23** [reply]
Not a mac. I only have two pdb files in the .zip.
How do you run oligomer and what problem do you face if I may ask?

**Comment by franke-biosaxs** [comment]
I asked because I'd observed issues in the past where, then on Mac, the .zip file contained extra files with meta-data that the OS secretly stashed there. It might be something similar here, but then again maybe not.

Do you have a local install of ATSAS-4.0.1? If yes, simply start the new [ATSAS Launcher](https://biosaxs-com.github.io/atsas/4.0.1/manuals/launcher.html) and follow the "Oligomer" wizard step by step. If you still get odd results I'd ask you to send me the data to have a look myself.

**Reply by ckos23** [reply]
Thank you. I will try that method and see if it gives better results. Also do you know how the masses are calculated and why they arent similar to the enzyme MW?
